src/AppBundle/Entity/VoteSettings.php

```diff
@@ -0,0 +1,180 @@
+<?php
+
+namespace AppBundle\Entity;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Component\Validator\Constraints as Assert;
+
+/**
+ * VoteSettings
+ *
+ * @ORM\Table()
+ * @ORM\Entity(repositoryClass="AppBundle\Entity\VoteSettingsRepository")
+ */
+class VoteSettings
+{
+    use GedmoTrait;
+
+    /**
+     * @var integer
+     *
+     * @ORM\Column(type="integer")
+     * @ORM\Id
+     * @ORM\GeneratedValue(strategy="AUTO")
+     */
+    private $id;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(type="string", length=255, nullable=true)
+     */
+    private $title;
+
+    /**
+     * @var integer
+     *
+     * @ORM\Column(name="vote_limits", type="integer", nullable=false)
+     */
+    private $voteLimits;
+
+    /**
+     * @var Project[]ArrayCollection
+     *
+     * @ORM\OneToMany(targetEntity="Project", mappedBy="voteSetting", cascade={"persist"})
+     */
+    private $project;
+
+    /**
+     * @var Location
+     *
+     * @ORM\ManyToOne(targetEntity="Location", inversedBy="voteSetting")
+     * @ORM\JoinColumn(name="location_id", nullable = true, referencedColumnName="id")
+     */
+    private $location;
```

Comment:
Location for the instance Project we can identification from VoteSetting for this project, because in Project VoteSetting AssertNotBlank. In feautire in insatns Project we can deprecated relation with location. I think this is good idea and very simple. Bit you can create like you want  

I think all VoteSetting we need save in one instance - location too